### PR TITLE
Add tutorial for custom objective to KerasTuner

### PR DESCRIPTION
**Pull request by haifeng-jin**
resolves https://github.com/keras-team/keras-tuner/issues/263

### CODE REVIEW

File: guides/keras_tuner/getting_started.py [hunk 1]
```diff
@@ -227,7 +227,60 @@ def build(self, hp):
...
+`keras.metrics.Metric`, the name is specified in the initializer. The
+`direction` should be either `'min'` (the lower the better) or `'max'` (the
+higher the better).
+
+"""
+
+import tensorflow as tf
+
+
+def mse(y_true, y_pred):
```
Comment (fchollet) [inline guides/keras_tuner/getting_started.py:259]: To be more generic, consider:

1. Using a `Metric` subclass instead of a function. For instance, AUC etc cannot be implemented as stateless functions, they must be `Metric` subclasses.
2. Pointing people to `train_step()` overriding if they need a metric that requires a different signature than `y_true, y_pred`. This is a fairly common question.

**Review by fchollet — APPROVED** [review]
LGTM, thanks! I applied some minor copyedits.
